docs/learn/resource-materials/glossary.md

```diff
@@ -1,34 +1,196 @@
+- **Distributed Ledger Technology (DLT):** This is a database architecture that allows owners of digital assets to transfer and document them from peer to peer. Each transfer in a DLT is stored as a record in a distributed ledger (database). This database is stored in all nodes of a network.
+- **DevNet:** The DevNet (developer network) is a pure test network to research and test a Tangle without Coordinator.
+- **Decay:** Both Mana and Pending Mana decay proportionally to its value, preventing Mana from growing indefinitely over time.
+- **Double-spending:** Double-spending is a threat to a digital money system, where the same single digital token can be spent more than once. Unlike physical money, a digital token consists of a digital file that can be duplicated or counterfeited.
+- **dRNG (Decentralized Random Number Generator):** This random number generator is required in Fast Probabilistic Consensus (FPC) to make the consensus model more resilient to attacks. In the case of conflicting transactions, the FPC votes on the transactions in question in several rounds. The threshold at which a node changes its mind in this vote is 50% +/- a small random deviation (using dRNG). In order to avoid stalemate or a specific outcome in the voting, this additional random component prevents potentially malicious nodes from influencing the voting process.
+- **Dust Protection (IOTA 1.5):** Someone who wants to harm IOTA could automatically send 1i for years to repeatedly recreated addresses, driving up the ledger’s memory requirements to the point where a full-node would eventually only run on large servers. In Chrysalis if you want to create a microtransaction (<1Mi) you need to activate dust on the receiving Address. That will alow you to receive a certain amount of dust. Also, the addresses with Colored Coins have to be tokenized. After the Coordicide, there will be another solution for IOTA 2.0.
+- **Directed Acyclic Graph (DAG):** A directed acyclic graph is DLT structure that consists of edges and vertices that never form a closed loop. The IOTA arcitecture "the Tangle" is a form of directed acyclic graph. 
+
+## E
+
+- **Eclipse attack:** A cyber-attack that aims to isolate a specific user rather than attack the entire network.
+- **Entropy:** In cryptology, this term represents a measure of the “disorder” in texts. Entropy is usually abbreviated with the Greek capital letter Η.
+- **ETH Virtual Machine:** A VM is what executes the Smart Contract code so that it runs deterministically. The language in which you run the code provides hooks to access the sandbox on which the VM runs. In the case of IOTA, this is the ISCP sandbox, which only provides access to IOTA tokens, thus consensus also runs on IOTA tokens. It is only the VM that IOTA works with, foreign tokens have nothing to do with a VM running the SC code, not the ETH VM, nor the Cartesi VM.
```

Comment:
This is not the correct definition of the EVM.

Reply:
The validator node in ISCP defines a sandbox and abstract interfaces for smart contract execution. As long as a Virtual Machine complies with the above, it can run on ISCP.
ISCP is therefore agnostic to the VM running the smart contract. Currently, ISCP is compatible with its native Wasm VM (compiled from Rust), the Ethereum Virtual machine (Solidity) future plans include the Cartesi Virtual machine (Linux), Golang and Move